main.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+from agent import get_graph
+from inputs import from_github
+from langchain_core.messages import HumanMessage
+from tools import get_pr_metadata
+
+from composio import Action
+
+import logging
+logging.getLogger().setLevel(logging.DEBUG)
+
+# TODO: 
+# 1. API_KEY
+# 2. Log
+# 3. RAG with langchain
+
+def main() -> None:
+    """Run the agent."""
+    owner, repo_name, pull_number = from_github()
+    print("\n*****************************\n",owner, repo_name, pull_number,"\n*****************************\n")
+
+    repo_path = f"/Users/lingyawen1/lyw/AGI/2_Agents/composio_test/{repo_name}"
```

Comment:
In `main.py`, use more descriptive variable names. For example, `repo_path` can be made more generic or configurable rather than hardcoding the path.